appdynamics/startAppDynamics.sh

```diff
@@ -204,6 +208,9 @@ f_initAppDynamicsEnabled() {
 # pre-requisites: 
 # =============================================================================
 
+# Optional env variable but needs to be defaulted if not set prior to any logging
+if [[ -z "${APPDYNAMICS_AGENT_HOME}" ]]; then APPDYNAMICS_AGENT_HOME="/apps/appdynamics/appAgent"; fi
```

Comment:
This line is fine, but an alternative way of doing that is:
APPDYNAMICS_AGENT_HOME="${APPDYNAMICS_AGENT_HOME:-/apps/appdynamics/appAgent}"